client/js/_author/reducers/items.js

```diff
@@ -1,42 +1,56 @@
-import _          from 'lodash';
-import guid       from '../../utils/guid';
-import genusTypes from '../../constants/genus_types';
+import _                       from 'lodash';
+import guid                    from '../../utils/guid';
+import { types, getQbankType } from '../../constants/genus_types';
 
 // Leave this empty. It will hold assessments by bank id. IE `state[someId] = {a_bank}`
 const initialState = {};
 
-
-function updateChoiceData(item) {
-  const newItem = _.cloneDeep(item);
-
-  if (!newItem.question) {
-    newItem.question = {
-      choices: {},
-    };
-    return newItem;
-  }
-
+function deserializeChoices(choices, answers) {
```

Comment:
I think in the future we should figure out how to put all of this in the middleware. For now, merging this is probably more important.